qdrant-landing/content/documentation/qdrant-alternatives.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Qdrant vs. Pinecone
+weight: 10
+---
+
+# Comparing Alternatives: Qdrant vs. Pinecone
+
+Qdrant might just be the right vector search engine for you - but before you even consider switching: every transition has its challenges, primarily understanding new terminologies and concepts. 
+That's where this guide comes in. It succinctly breaks down the key differences between Qdrant and other vector databases and engines, paving the way for a smoother, more informed transition:
+
+| Feature                         | Pinecone                      | Qdrant                                       | Comments                                                 |
+|---------------------------------|-------------------------------|----------------------------------------------|----------------------------------------------------------|
+| Deployment Modes                | SaaS-only                     | Local, on-premise, Cloud                     | Qdrant offers more flexibility in deployment modes       |
+| Supported Technologies          | Python, JavaScript/TypeScript | Python, JavaScript/TypeScript, Rust, Go      | Qdrant supports a broader range of programming languages |
+| Performance (e.g., query speed) | TnC Prohibit Benchmarking     | [Benchmark result](/benchmarks/)             | Compare performance metrics                              |
+| Pricing                         | Starts at $70/mo              | Free and Open Source, Cloud starts at $25/mo | Pricing as on May 2023                                   |
+
+## Pinecone Overview
+
+Pinecone is a SaaS-only vector database written in Rust. Since it's not Open Source, we couldn't include it in our 
+[benchmarks](/benchmarks/), but that's a popular tool among the users who decide to switch to Qdrant. 
+
+## Deployment modes
+
+While Qdrant offers multiple ways of deployment, including local mode, on-premise, and [Qdrant Cloud](https://cloud.qdrant.io/), 
+Pinecone is available in a SaaS-only model. Thus, even your development environment has to connect with the cloud services,
+while Qdrant offers you some local modes as well.
```

Comment:
Strong recommend: Example code which shows how Qdrant persists vectors across sessions. Without using `:memory:`, preferably with the Python Client.

Reply:
In my opinion, we should skip any code examples in the comparisons to the alternatives. There should be rather some bullet points, for those who are familiar with the competitors. We can, however, link some other pages in the docs.

Reply:
agreed, this isn't the type of doc where people read code